code/modules/research/designs.dm

```diff
@@ -175,6 +175,46 @@ other types of metals and chemistry for reagents).
 	id = "jackhammer"
 	req_tech = list("materials" = 6, "powerstorage" = 6, "engineering" = 5, "magnets" = 6)
 	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 6000, MAT_GLASS = 2000, MAT_SILVER = 2000, MAT_DIAMOND = 6000)
+	build_path = /obj/item/weapon/pickaxe/drill/jackhammer
+	category = list("Mining Designs")
+
+/datum/design/superaccelerator
+	name = "Super-Kinetic Accelerator"
+	desc = "An upgraded version of the proto-kinetic accelerator, with superior damage, speed and range."
+	id = "superaccelerator"
+	req_tech = list("materials" = 5, "powerstorage" = 4, "engineering" = 4, "magnets" = 4, "combat" = 3)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 8000, MAT_GLASS = 1500, MAT_SILVER = 2000, MAT_URANIUM = 2000)
+	build_path = /obj/item/weapon/gun/energy/kinetic_accelerator/super
+	category = list("Mining Designs")
+
+/datum/design/hyperaccelerator
+	name = "Hyper-Kinetic Accelerator"
+	desc = "An upgraded version of the proto-kinetic accelerator, with eve more superior damage, speed and range."
+	id = "hyperaccelerator"
+	req_tech = list("materials" = 6, "powerstorage" = 6, "engineering" = 5, "magnets" = 6, "combat" = 4)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 8000, MAT_GLASS = 1500, MAT_SILVER = 2000, MAT_GOLD = 2000, MAT_DIAMOND = 2000)
+	build_path = /obj/item/weapon/gun/energy/kinetic_accelerator/hyper
+	category = list("Mining Designs")
+
+/datum/design/superresonator
+	name = "Upgraded Resonator"
+	desc = "An upgraded version of the resonator that allows more fields to be active at once."
+	id = "superresonator"
+	req_tech = list("materials" = 4, "powerstorage" = 3, "engineering" = 3, "magnets" = 3)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 4000, MAT_GLASS = 1500, MAT_SILVER = 2000, MAT_URANIUM = 2000)
+	build_path = /obj/item/weapon/resonator/upgraded
+	category = list("Mining Designs")
+
+/datum/design/jackhammer

```

Comment:
I am confused, this appears to already have been defined on line 172. Please check for duplication, and remove the extra one.